README.md

```diff
@@ -84,13 +84,22 @@ The CloudBees CD plugin enables you to perform post-build actions for your Jenki
 
 ## Calling CloudBees CD REST API
 
-The CloudBees CD plugin allows you to make calls to CloudBees CD's REST API. You can make for post-build actions and as pipeline steps.
+The CloudBees CD plugin allows you to make calls to CloudBees CD's REST API for post-build actions and as pipeline steps. For more information on using the CloudBees CD REST API, refer to [Use the CloudBees CD/RO RESTful API](https://docs.cloudbees.com/docs/cloudbees-cd-api-rest/latest/).  
 
-To configure a calls to CloudBees CD's REST API:
+To configure calls to CloudBees CD's REST API:
 1. Navigate to your job's **Configuration** > **Post-build Actions** menu.
 2. Select **Add post-build action**.
 3. In the filter field, enter `CloudBees CD - Call REST API`. 
-4. Complete the fields with your configuration details. Select the **?** icon to view field information in a new dialog. 
+4. Complete the fields with your configuration details. Select the **?** icon to view field information in a new dialog.
+    > **_IMPORTANT:_** For API parameters that include the phrase,  "`request=sendReportingData` must appear in your query string" in their description, you must include `request=sendReportingData` in the **Path URL** field. For example:
```

Comment:
@jsmith-cb , could you please confirm with CD team cases when request= should be used with different values, I assume request=sendReportingData is not single case

Reply:
@ovasilko, 
I discussed this with Valeriy, and we agreed on the wording in [a1ef527](https://github.com/jenkinsci/electricflow-plugin/pull/356/commits/a1ef5274d56b1bb7721258cb3103f4628461454a). Please let me know if this addresses your concern.